tests/unit/lms/views/test_application_instance.py

```diff
@@ -68,6 +68,24 @@ def test_it_sets_canvas_sections_enabled(
             == canvas_sections_enabled
         )
 
+    @pytest.mark.parametrize(
+        "developer_key,canvas_groups_enabled",
+        [("test_developer_key", True), ("", False)],
+    )
+    def test_it_sets_canvas_groups_enabled(
```

Comment:
Test is copy pasted from the section one. I think we should test both settings independently.